src/azure-cli/azure/cli/command_modules/iot/_help.py

```diff
@@ -768,60 +768,80 @@
+    IoT Central documentation is available at https://aka.ms/iotcentral-documentation
+    Comprehensive IoT Central data-plane functionality is available in the Azure IoT CLI Extension. For more info on how to install visit https://github.com/Azure/azure-iot-cli-extension
 """
 
 helps['iot central app'] = """
 type: group
 short-summary: Manage IoT Central applications.
+long-summary: Create, delete, view, and update your IoT Central apps.
 """
 
 helps['iot central app create'] = """
 type: command
 short-summary: Create an IoT Central application.
 long-summary: |
-    For an introduction to IoT Central, see https://docs.microsoft.com/azure/iot-central/.
-    The F1 Sku is no longer supported. Please use the ST2 Sku (default) for app creation.
-    For more pricing information, please visit https://azure.microsoft.com/pricing/details/iot-central/.
+    An IoT Central app is a cloud-based application that you can use to manage your IoT devices.
+    For more information on deploying an IoT Central app via CLI or a list of available arguments, visit https://aka.ms/iotcentral-manage-cli
 examples:
-  - name: Create an IoT Central application in the standard pricing tier ST2, in the region of the resource group.
+  - name: Create an IoT Central application in the region of the resource group using the default pricing SKU.
     text: >
-        az iot central app create --resource-group MyResourceGroup --name my-app-resource --subdomain my-app-subdomain
-  - name: Create an IoT Central application with the standard pricing tier ST2 in the 'westus' region, with a custom display name, based on the iotc-pnp-preview template.
+        az iot central app create -n my-first-iot-app -g MyResourceGroup -s my-iot-app -l unitedstates
+  - name: Create an IoT Central application the in the 'United States' region with the 'ST1' pricing plan and an industry template.
     text: >
-        az iot central app create --resource-group MyResourceGroup --name my-app-resource-name --sku ST2 --location westus --subdomain my-app-subdomain --template iotc-pnp-preview --display-name 'My Custom Display Name'
+        az iot central app create -n my-first-iot-app -g MyResourceGroup -s my-iot-app -l unitedstates -p ST1 -t iotc-distribution
 """
 
 helps['iot central app delete'] = """
 type: command
 short-summary: Delete an IoT Central application.
+long-summary: |
+    Permanently delete your IoT Central app.
+    -Warning- This operation is irreversible. Once you've deleted your app, you cannot restore it.
 examples:
-  - name: Delete an IoT Central application.
-    text: az iot central app delete --name MyIoTCentralApplication --resource-group MyResourceGroup
+  - name: Deletes an IoT Central app called 'my-first-iot-app'.
```

Comment:
Please use the first person voice~
```suggestion
  - name: Delete an IoT Central app called 'my-first-iot-app'.
```